src/tests/ftest/container/container_redundancy.yaml

```diff
@@ -0,0 +1,36 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+    - server-F
+  test_clients:
+    - client-G
+timeout: 360
+server_config:
+  name: daos_server
+  servers:
+    targets: 2
```

Comment:
Any idea why you are reducing the number of targets from default 8 to 2?

Reply:
"servers targets: 2" is consisted with rebuild/cascading_failures.yaml and rebuild/read_array.yaml.
We can upgrade it to 8 if it is appropriate and needed.

Reply:
I would like to leave the targets as default and see how things work. Let's not set the targets to 2.

Reply:
Please see the update.

Reply:
After ran tests with targets:2 and targets:8, VM is only allow 2 targets per server, so have to reduce targets to 2.
Targets:8 have also been tested on local boro hosts, and working fine.